src/crewai/cli/organization/main.py

```diff
@@ -0,0 +1,63 @@
+from rich.console import Console
+from rich.table import Table
+
+from crewai.cli.command import BaseCommand, PlusAPIMixin
+from crewai.cli.config import Settings
+
+console = Console()
+
+class OrganizationCommand(BaseCommand, PlusAPIMixin):
+    def __init__(self):
+        BaseCommand.__init__(self)
+        PlusAPIMixin.__init__(self, telemetry=self._telemetry)
+
+    def list(self):
+        try:
+            response = self.plus_api_client.get_organizations()
+            response.raise_for_status()
+            orgs = response.json()
+            
+            if not orgs:
+                console.print("You don't belong to any organizations yet.", style="yellow")
+                return
+
+            table = Table(title="Your Organizations")
+            table.add_column("Name", style="cyan")
+            table.add_column("Handle", style="green")
+            for org in orgs:
+                table.add_row(org["name"], org["uuid"])
+            
+            console.print(table)
+        except Exception as e:
+            console.print(f"Failed to retrieve organization list: {str(e)}", style="bold red")
+            raise SystemExit(1)
+
+    def switch(self, org_handle):
+        try:
+            response = self.plus_api_client.get_organizations()
+            response.raise_for_status()
+            orgs = response.json()
+            
+            org = next((o for o in orgs if o["uuid"] == org_handle), None)
+            if not org:
+                console.print(f"Organization with handle '{org_handle}' not found.", style="bold red")
+                return
+            
+            settings = Settings()
+            settings.org_name = org["name"]
+            settings.org_uuid = org["uuid"]
+            settings.dump()
+            
+            console.print(f"Successfully switched to {org['name']} ({org['uuid']})", style="bold green")
+        except Exception as e:
+            console.print(f"Failed to switch organization: {str(e)}", style="bold red")
+            raise SystemExit(1)
+
+    def current(self):
+        settings = Settings()
+        if settings.org_uuid:
+            console.print(f"Currently logged in to organization {settings.org_name} ({settings.org_uuid})", style="bold green")
+        else:
```

Comment:
The default org is set by the server - if we call the API we should get a response back with `current_organization`, which uses the `#first` strategy.